code/optimize/src/main/java/com/adobe/marketing/mobile/optimize/Optimize.java

```diff
@@ -88,6 +88,44 @@ public static void updatePropositions(
             @Nullable final Map<String, Object> xdm,
             @Nullable final Map<String, Object> data,
             @Nullable final AdobeCallback<Map<DecisionScope, OptimizeProposition>> callback) {
+        final long defaultTimeout = OptimizeConstants.EDGE_CONTENT_COMPLETE_RESPONSE_TIMEOUT;
+        updatePropositionsInternal(decisionScopes, xdm, data, defaultTimeout, callback);
```

Comment:
This will break backward compatibility right?

Reply:
@jfkhoury, Thanks for reviewing this PR
Could you please elaborate more how this will impact our backward compatibility

Reply:
As per my understanding Since I'm not changing any existing public `updatePropositions()` overloaded method/API arguments, I don't feel this will impact any of our existing usecase
Having said that, one proof of that is i didn't made any change to any `updatePropositions()` calls inside our existing unit tests
Feel free to ping me if you have other POV